wAbout/Enterprise-Login-Setup.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Enterprise Login Setup
+layout: custom
+keywords: [login, ribbon, credentials, accounts, Enterprise, setup, oidc, auth, code]
+description: 
+---
+* * *
+
+## Overview
+
+Interject supports federated logins for enterprise allowing a single sign on (SSO) experience for Interject users within an organization. Currently, [Open ID Connect](https://openid.net/developers/how-connect-works/){:target="_blank"}{:rel="noopener noreferrer"} (OIDC) is the primary protocol Interject supports but more federated authentication systems are planned to be supported, including [Security Assertion Markup Language](https://www.oasis-open.org/standard/saml/){:target="_blank"}{:rel="noopener noreferrer"} (SAML).
+
+There are other identity provider services on the cloud that support OIDC. The following are especially easy for Interject to work with, but any system supporting OIDC will work:
+
+* [Azure AD](https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-protocols-oidc){:target="_blank"}{:rel="noopener noreferrer"}
+* [Okta](https://developer.okta.com/docs/concepts/auth-servers/){:target="_blank"}{:rel="noopener noreferrer"}
+* [Auth0](https://auth0.com/docs/get-started/authentication-and-authorization-flow){:target="_blank"}{:rel="noopener noreferrer"}
+
+<blockquote class=highlight_note>
+<b>Note:</b> Users will need Interject version 2.5.0.14 or above to use federated logins. Interject can set it up where federated users and users on the existing system exist together, so we can migrate/update users individually if that makes it smoother to upgrade.
```

Comment:
just a small thing here but having exist and existing so close together in the same sentence is a little awkward. Maybe this works better: "Interject can be set up to accommodate federated and non-federated users existing on the same system..."

Reply:
Yes that sounds better, I can change